libcontainer/cgroups/fscommon/rdma.go

```diff
@@ -0,0 +1,121 @@
+// +build linux
+
+package fscommon
+
+import (
+	"bufio"
+	"errors"
+	"math"
+	"strconv"
+	"strings"
+
+	"github.com/opencontainers/runc/libcontainer/cgroups"
+	"github.com/opencontainers/runc/libcontainer/configs"
+	"golang.org/x/sys/unix"
+)
+
+// parseRdmaKV: parses raw string to RdmaEntry.
+func parseRdmaKV(raw string, entry *cgroups.RdmaEntry) error {
+	var value uint64
+	var err error
+
+	parts := strings.SplitN(raw, "=", 3)
```

Comment:
SplitN 2 here?

Reply:
Better to use Splitn 3, so that the code is able to catch len(parts) > 2

Reply:
@fuweid This is needed resolving this,